### DISCUSSION TITLE
Windows 10 installation of rabbitmq-server-3.11.0 installed successfully, failed to start

### DISCUSSION THREAD

**Discussion by noobwu** [rabbitmq/rabbitmq-server#6089] [Other] [answered]
Windows OS information is as follows：
  - OS Version: Windows 10 IoT Enterprise Edition LTSC
  - OS  Version number: 21H2
  - OS Internal Version: 19044.2006

erlang：otp_win64_25.1.1.exe
rabbitmq-server-3.11.0.exe

erl_crash.dump：
[erl_crash.zip](https://github.com/rabbitmq/rabbitmq-server/files/9752784/erl_crash.zip)

**Comment by michaelklishin** [comment]
Please attach server logs or any node output if there are none, also  see https://rabbitmq.com/windows-quirks.html to learn about potential environment issues specific to Windows

**Comment by lukebakken** [comment]
```
=erl_crash_dump:0.5
Tue Oct 11 15:37:56 2022
Slogan: init terminating in do_boot ({undef,[{rabbit,boot,[],[]},{init,start_em,1,[{_},{_}]},{init,do_boot,3,[{_},{_}]}]})
System version: Erlang/OTP 25 [erts-13.1.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [jit:ns]
```

Please provide the **exact** commands you used to install Erlang and RabbitMQ, as well as the output of those commands. If you just double-clicked both installers and followed the defaults, please say so.

**Comment by noobwu** [comment]
I installed it by default by double-clicking on the installer

**Reply by lukebakken** [reply]
Please see the steps I used here and compare with yours:

https://groups.google.com/g/rabbitmq-users/c/rcDdmeG-fvA/m/bvkjjw7eAAAJ

I don't have access to "Windows 10 IoT Enterprise Edition" but I can't imagine it's that much different as far as RabbitMQ is concerned.

**Comment by michaelklishin** [comment]
`undef` for `rabbit:boot/0` means something is off with the way the code path of the node is set up. 

See a number of [known Windows-specific issues](https://www.rabbitmq.com/windows-quirks.html). You can have more than one Erlang version installed or use a directory with non-ASCII characters (support for which has been somewhat improved in `3.11.1` but we still highly recommend against using them in installation and node data directory paths).

**Comment by noobwu** [comment]
Thank you very much! I'll try it sometime.

**Comment by mgrbes** [comment]
What's your installation folder destination?
I've had exact same problem it installs but it won't run it shuts down immediately, my folder where to install was on D drive.
After some time when you run installation again(reinstall it not uninstall) it works perfectly, after checking installation process it installs plugins by default on C drive and not on D where i put the installation folder. So either run the installer twice or copy plugins from C drive to your RabbitMQ folder on D drive.

**Reply by mgrbes** [reply]
[rabbit mq differences.pdf](https://github.com/rabbitmq/rabbitmq-server/files/9877408/rabbit.mq.differences.pdf)

**Comment by lukebakken** [comment, marked as answer]
@mgrbes @noobwu please see this issue: https://github.com/rabbitmq/rabbitmq-packaging/issues/28

You can work around it by specifying the installation directory on the command line -

https://github.com/rabbitmq/rabbitmq-packaging/issues/28#issuecomment-1291171268

```
.\rabbitmq-server-3.11.2.exe /D=D:\Program Files\RabbitMQ Server
```

Note that `/D` _must_ be the last argument and that you should **not** double-quote the argument, even if it has spaces.

**Reply by noobwu** [reply]
Thank you very much!

**Reply by noobwu** [reply]
PS E:\Software> .\rabbitmq-server-3.11.2.exe /D=E:\Program Files\RabbitMQ Server
After installation, RabbitMQ can be started normally according to the instructions you provided.

**Reply by lukebakken** [reply]
Thank you for following up.

**Reply by noobwu** [reply]
Learn more from you.
